pkgs/applications/editors/vim/plugins/deprecated.json

```diff
@@ -7,10 +7,6 @@
         "date": "2021-12-07",
         "new": "cmp-tmux"
     },
-    "taskwarrior": {
-        "date": "2024-08-13",
-        "new": "taskwarrior3 or taskwarrior2"
-    },
```

Comment:
Drop 72447c33ed32c437992496936848fc9b0e7a6b26 please, it is useless (or squash it with 28bc9ba3c6158ed3d92242184e2e5468168ff6c2). The functionality, that is done by this commit is broken and will be fixed by #336137

Reply:
Unfortunately, yes. It is broken